lib/utils/dataflow/src/index.js

```diff
@@ -380,9 +432,10 @@ const mapper = (mapfn, opt) => {
 
   // Map an input doc to an output doc, store both the input and output and
   // pass the output to the configured sink service
-  /* eslint complexity: [1, 6] */
+  /* eslint complexity: [1, 10] */
```

Comment:
Yeah you're right. I tried to split the code, but I was unsure on how to split it and I thought that I should get feedback first.